### PR TITLE
Api config

### PR DESCRIPTION
**Pull request by Reedyuk**
Whats the best way to get the bearer token? 
I have done all the rest.

### CODE REVIEW

File: admin/client/App/screens/Api/actions.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+export function clearCache () {
+	//api to be using token and delete
+	return (dispatch) => {
+		dispatch({
+			type: CLEAR_CACHE,
+		});
+		xhr({
+			method: "delete",
+		    headers: {
+		        "Authorization": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbElkIjoiYW5kcmV3QGtpZGRvb2FwcC5jb20iLCJyb2xlIjoiYWRtaW4iLCJpYXQiOjE1MTA4NDE1NDN9.ttdXJ6N0O1nX68RE7h319NE1f44XWl3k5h3ulDkF0F0"
```
Comment (Reedyuk) [inline admin/client/App/screens/Api/actions.js]: how can i generate the bearer token?

Comment (atanmayprdxn) [reply admin/client/App/screens/Api/actions.js]: @Reedyuk : We have two APIs one for cms and one for the app.
If you want the cms token you won't need to generate it. Just hardcode it.
If you want the web app token to use the signup API for the app.

Comment (Reedyuk) [reply admin/client/App/screens/Api/actions.js]: great i can make that an environment variable then, what is the cms token?

Comment (Reedyuk) [reply admin/client/App/screens/Api/actions.js]: slack me the answer, not on here.

**Pull request merged**